xapian-letor/feature/colltfcolllenfeature.cc

```diff
@@ -94,7 +94,7 @@ CollTfCollLenFeature::get_values() const
     }
     values.push_back(value);
     value = 0;
-    coll_len_iterator = collection_length.find("body");
+    coll_len_iterator = collection_length.find("whole");
```

Comment:
If you've made changes to more than just the tests, then the commit message should reflect that you've also fixed a bug / changed behaviour. Even better would be to separate that out — add the tests that pass without this change as one commit, then have a separate commit for this change and the test(s) that exercise it. (The commit message can then explain in more detail what's going on — as it stands it's very hard for anyone to review this, because there's no indication why this change is appropriate.)

Reply:
sorry, will create new commits for different changes.